src/App.vue

```diff
@@ -517,12 +516,56 @@
         </div>
       </div>
     </div>
+    <div class="py-5">
+      <i-calendar
+        v-model="computedStuffingDate"
```

Comment:
computedStuffingDate should updated based on i-calendar selected date

Reply:
ok